backend/server.py

```
@@ -27,6 +27,40 @@ def _is_writable(stream):
+
+    def write(self, data):
+        try:
+            return self._base.write(data)
+        except (BrokenPipeError, OSError):
+            return len(data)
+
+    def flush(self):
+        try:
+            self._base.flush()
+        except (BrokenPipeError, OSError):
+            pass
```

Ruff (0.15.20)

[warning] Use `contextlib.suppress(BrokenPipeError, OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(BrokenPipeError, OSError): ...`

(SIM105)